terraform/copy/copy_dir.go

```diff
@@ -0,0 +1,146 @@
+package copy
```

Comment:
Do you need these codes? I have confirmed that the `CopyDir` is referenced from the `getmodules` package, but it seems that `getWithGoGetter` and `FetchPackage` that use it are not used from there either.

Whenever possible, I recommend that you remove unused code. This should reduce the code under the `terraform` directory that must be added with this change.

Reply:
Removed `package copy`. I get the desire to reduce LOC from Terraform but the flip side is that modifying packages to remove dependencies on others makes it harder to copy in the latest version of a package from upstream.